tfadd/internal/tune_tpl.go

```diff
@@ -80,7 +82,10 @@ func tuneForBlock(rb *hclwrite.Body, sch *tfpluginschema.Block, parentAttrNames
 		if diags.HasErrors() {
 			return fmt.Errorf(`parsing HCL expression %q: %s`, string(attrValLit), diags.Error())
 		}
-		aval, diags := dexpr.Value(nil)
+		aval, diags := dexpr.Value(&hcl.EvalContext{Functions: map[string]function.Function{
+			"jsondecode": stdlib.JSONDecodeFunc,
```

Comment:
Is this needed?

Reply:
Yes. `dexpr.Value()` need such Context of Functions to parse the `dexpr` with function call. or it will cause error like `Function calls not allowed`

Reply:
I mean we only used `jsonencode`, not `jsondecode`?

Reply:
`jsonencode` is enough.